app/controllers/user_notifications_controller.rb

```diff
@@ -21,4 +21,11 @@ def delete
     ).destroy!
     render nothing: true
   end
+
+  def unsubscribe
+    user_notif = UserNotification.find_by(token: params[:token])
```

Comment:
Seems like you're unsubscribing the users from all notifications here. Maybe it's best if we can provide the option to either only unsubscribe from the current notification type, or unsubscribe from all emails.

It can be
- two links
- link to a view that unsubscribes the user, then have a button for the user to unsubscribe from all

Reply:
2 links make it easier for them, i guess